### ISSUE TITLE
Panic invalid token every 24 hours

### ISSUE THREAD

**Issue by AlexRichter** [martinohansen/ynabber#97]
I have ynabber working well except every 24 hours a panic is caused getting account metadata, on automatic restart of the container the next run will work as normal. I have it set to run every 6 hours and it panics on the same run every single day. 

```
panic: reading: failed to get account metadata: APIError 401 {\summary\:\Invalid token\,\detail\:\Token is invalid or expired\,\status_code\:401}

goroutine 1 [running]:

main.main()

/go/src/app/cmd/ynabber/main.go:65 +0x6c5
```

**Comment by martinohansen** [comment]
I have the same, should be fixed in the next lib update.

**Comment by martinohansen** [comment]
@AlexRichter finally came around to merge the fix, should work in [v0.8.1](https://github.com/martinohansen/ynabber/releases/tag/v0.8.1)
